package_sprint.sh

```diff
@@ -10,9 +10,7 @@ STAGING_DIR=$STAGING_DIR_BASE/$STAGING_DIR_NAME
 FINAL_TARGET_DIR=/tmp
```

Comment:
As suggested by @mradcliffe (I think) let's change this file to package_drupal_script.sh and update the README to match.